lambdas/process-cri-callback/src/main/java/uk/gov/di/ipv/core/processcricallback/ProcessCriCallbackHandler.java

```diff
@@ -284,8 +285,9 @@ private JourneyResponse getJourneyResponse(CriCallbackRequest callbackRequest)
 
         // Retrieve, store and check cri credentials
         var accessToken = criApiService.fetchAccessToken(callbackRequest, criOAuthSessionItem);
-        // Temporarily logging DWP KBV token for integration debugging
-        if (callbackRequest.getCredentialIssuer().equals(DWP_KBV)
+        // Temporarily logging staging token for debugging purposes
+        if ((callbackRequest.getCredentialIssuer().equals(DWP_KBV)
+                        || callbackRequest.getCredentialIssuer().equals(NINO))
```

Comment:
I want to be able to reproduce NINO CRI VC requests locally too, as NINO CRI weirdly seems to require a Content-Type set to empty string which doesn't seem right